pyzeebe/exceptions/pyzeebe_exceptions.py

```diff
@@ -20,3 +20,6 @@ class DuplicateTaskType(PyZeebeException):
     def __init__(self, task_type: str):
         super().__init__(f"Task with type {task_type} already exists")
         self.task_type = task_type
+
+class MaxConsecutiveTaskThread(PyZeebeException):
```

Comment:
@JonatanMartens I'm introducing a new exception here. However, this doesn't necessarily sound like an error. PEP 8 suggests to add ...Error to it's name - should we start doing that?

https://www.python.org/dev/peps/pep-0008/#exception-names

I.e. `raise MaxConsecutiveTaskThreadError("Too many task thread errors...")`

Reply:
Yes, I believe `MaxConsecutiveTaskThreadError` is preferable.

Reply:
Addressed in e68219b6c78ef23668a61585c7f5dceeddf95bd5